macros/utils/cross_db_utils/current_timestamp.sql

```diff
@@ -0,0 +1,38 @@
+{% macro current_timestamp() -%}
```

Comment:
Is this just copying from dbt utils? 
If so we need to credit them here and add the licensing